cmd/db.go

```diff
@@ -0,0 +1,95 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	db "github.com/cosmos/cosmos-db"
+)
+
+const (
+	rocksDBMagic  = "\xf7\xcf\xf4\x85\xb7\x41\xe2\x88"
+	pebbleDBMagic = "\xf0\x9f\xaa\xb3\xf0\x9f\xaa\xb3" // 🪳🪳
```

Comment:
what does the comment mean?

Reply:
each of the magic strings is 8 bytes, used to identify the format of the database.
in pebble's case, it's literally the representation of 2 cockroach emojis (pebble is made by cockroach db)

```
$ echo -n 🪳🪳 | hexdump -C
f0 9f aa b3 f0 9f aa b3
```